lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -774,11 +774,10 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     No = false,
     Yes = true
   };
-
-  /// Returns \c true if anything was printed.
-  bool printAvailability(
-      const Decl *D,
-      PrintLeadingSpace printLeadingSpace = PrintLeadingSpace::Yes) {
+ 
+    /// Returns \c true if anything was printed.
+  bool printAvailability(const Decl *D, PrintLeadingSpace printLeadingSpace =
+                                            PrintLeadingSpace::Yes) {
```

Comment:
Nitpick: I think the existing formatting (double-indented wrapping) is preferred over splitting on the `=`. (Ultimately it's whatever `clang-format` says, though.)

Reply:
This is what `clang-format` says. Do you want me to change?

Reply:
Huh. Guess not, sorry for the noise.